examples/gno.land/p/demo/json/license/LICENSE.txt

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
> `mv examples/gno.land/p/demo/json/license/LICENSE.txt examples/gno.land/p/demo/json/LICENSE`

right now we treat sub dirs as an independent realm/pkg (See #1024). 
Please move this to the root dir of the package.

also, please rename this file to `LICENSE`. As for now we only support these 3 file names (other than .gno file):
https://github.com/gnolang/gno/blob/b93f5901f4ea78a666af33b4490ecc0c0567da98/gnovm/pkg/gnolang/nodes.go#L1110-L1114